packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEdit.tsx

```diff
@@ -44,7 +44,8 @@ export const StocktakeLineEdit: FC<StocktakeLineEditProps> = ({
 
   const onNext = async () => {
     await save(draftLines);
-    if (nextItem) setCurrentItem(nextItem);
+    if (mode === ModalMode.Update && nextItem) setCurrentItem(nextItem);
+    else if (mode === ModalMode.Create) setCurrentItem(null);
```

Comment:
umm.. not sure but made it the same at least?

Reply:
👍  very nice user experience!